person.rb

```diff
@@ -1,21 +1,24 @@
+require './rentals'
+
```

Comment:
- [ ] Please add the Rental class with its associations because it looks like you forgot it

Reply:
@hiromataba the rental class have been added with the add rentals method.

Reply:
- [ ] @V0pe  I cannot see those classes in your files, where are they located please ?


![image](https://user-images.githubusercontent.com/75126481/180187035-07c4ffc2-a543-4d42-bfce-1bd2a2f3f7dd.png)

Reply:
@hiromataba Apologies, It wasnt saved
Can you go over it again now

Reply:
@V0pe  Sorry but I cannot check that now because I have already submitted my review but the next reviewer can see it I believe

Reply:
@hiromataba can i call, i have some questions

Reply:
@V0pe  I wish it could be possible but I am currently reviewing another project now